src/adapters/pnp/PnpAgent.c

```diff
@@ -105,7 +105,7 @@ static const char g_productInfoTemplate[] = "Azure OSConfig %d;%s "
     "\"product_vendor\"=\"%s\"&\"product_name\"=\"%s\")";
 static char g_productInfo[DEVICE_PRODUCT_INFO_SIZE] = {0};
 
-OSCONFIG_LOG_HANDLE GetLog()
+OsConfigLog* GetLog()
```

Comment:
With the proposed way to keep existing names, files like this won't be changed and minimize the need for wider validation. For example, there is no CI for the PnP Agent where this code is. We had CI, we had an E2E test, that was then cut/removed. So making changes in anything that gets consumed here comes with a risk and  with need of extensive manual validation.